eng/common/js-sdk-release-tools/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@azure-tools/js-sdk-release-tools": "2.14.9"
```

Comment:
If this is only used in the JS repo then we should just put it there and not under eng/common.